File: core/src/rustyice/game/tiles/GlassTile.java
package rustyice.game.tiles;

import com.badlogic.gdx.graphics.Color;
import com.badlogic.gdx.graphics.g2d.Sprite;
import rustyice.game.physics.RectWallComponent;

public class GlassTile extends Tile {

    public GlassTile() {
        super(true, false);
        setTilePhysics(new RectWallComponent());
    }

    @Override
    public void init() {
        super.init();
        Sprite sprite = new Sprite(getResources().box);
        sprite.setColor(new Color(0, 0.1f, 0.1f, 0.5f));
        setSprite(sprite);
    }
}

File: core/src/rustyice/game/tiles/GlassTile.kt
package rustyice.game.tiles

import com.badlogic.gdx.graphics.Color
import com.badlogic.gdx.graphics.g2d.Sprite
import rustyengine.RustyEngine
import rustyice.game.physics.RectWallComponent
import rustyengine.resources.Resources

class GlassTile() : Tile(true, false) {

    override fun init() {
        super.init()

        val sprite = Sprite(RustyEngine.resorces.box)
        sprite.color = Color(0f, 0.1f, 0.1f, 0.5f)

        this.sprite = sprite
    }

    init {
        tilePhysics = RectWallComponent()
    }
}
